### ISSUE TITLE
[Adaptive Cards iOS - Calendar Reminder]: List items under 'Snooze for' dropdown list is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6964]
https://microsoft.visualstudio.com/OS/_workitems/edit/33755242
